cmd/root.go

```diff
@@ -385,7 +385,7 @@ octosql "SELECT * FROM plugins.plugins"`,
 
 		switch output {
 		case "live_table", "batch_table":
-			var limit *int
+			var limit *int64
```

Comment:
let me know if that's OK for you, I think we could safely use the 32 bit here as before. Most of the PR here is to adjust for that type change.